src/daemon/osd_scenarios/osd_volume_activate.sh

```diff
@@ -95,18 +95,20 @@ function osd_volume_activate {
     osd_volume_simple
   fi
 
+  if [[ ${OSD_DMCRYPT} -eq 1 ]]; then
+    umount_lockbox
+  fi
+
```

Comment:
this should be done only if OSD_VOLUME_TYPE==simple so probably need to integrate this statement after the osd_volume_simple call